Fix wrong rotation in GOMove

Magnifikator opened the pull request.

Rochet2 commented:
Magnifikator fixing everything :D
I made a new addon that does not require server side edits. Interested in trying it out?
It works only on TC 3.3.5 i guess. it uses TC commands to do the heavy lifting.
By the way. Is GOMove used alone or editing with other people?

Magnifikator commented:
I have modified GOMove heavily on our server ;-) 
On our server GOMove can be used by all players, but only in specific custom zones and only with useful gameobjects like plants, furnitures, houses, decorative objects and so on. We expanded therefore our databases with thousands of objects and added a custom MPQ.
For some gameobjects I added a autoturn and a autosize function. When adding plants like grasses, bushes and trees, they are added with a random modification of angle and size so it look more natural :)
GM's of course can use GOMove everywhere and for all GOs.

Beside of that players also can add some NPCs like animals or humanoid creatures.

Unfortunately today it seems nobody is interested in WOW anymore and the server is mostly empty.

Rochet2 commented:
I see.
